test_conformance/spirv_new/CMakeLists.txt

```diff
@@ -27,6 +26,7 @@ set(${MODULE_NAME}_SOURCES
   test_op_vector_extract.cpp
   test_op_vector_insert.cpp
   test_op_vector_times_scalar.cpp
+  test_no_integer_wrap_decoration.cpp
```

Comment:
Sorry one more nit: it would be nice to keep this list in alphabetical order.

Reply:
Ah, I didn't realize they were in alphabetical order previously.  Fixed!